components/remote_settings/src/client.rs

```diff
@@ -61,22 +64,16 @@ impl Client {
     /// bucket, and collection defined by the [ClientConfig] used to generate
     /// this [Client]. This function will return the raw network [Response].
     pub fn get_records_raw(&self) -> Result<Response> {
-        let path = format!(
-            "v1/buckets/{}/collections/{}/records",
-            &self.bucket_name, &self.collection_name
-        );
-        self.make_request(path)
+        self.get_records_raw_with_options(&GetItemsOptions::new())
     }
 
     /// Fetches all records that have been published since provided timestamp
     /// for a collection that can be found in the server, bucket, and
     /// collection defined by the [ClientConfig] used to generate this [Client].
     pub fn get_records_since(&self, timestamp: u64) -> Result<RemoteSettingsResponse> {
-        let path = format!(
-            "v1/buckets/{}/collections/{}/records?_since={}",
-            &self.bucket_name, &self.collection_name, timestamp
-        );
-        let resp = self.make_request(path)?;
+        let resp = self.get_records_raw_with_options(
+            GetItemsOptions::new().gt("last_modified", timestamp.to_string()),
+        )?;
```

Comment:
According to https://docs.kinto-storage.org/en/latest/api/1.x/filtering.html#polling-for-changes, `_since` is an alias for `gt_last_modified`. 😊